environments/values.yaml

```diff
@@ -181,9 +202,25 @@ applications:
   # -- Enable the times-square application
   times-square: false
 
+  # -- Enable the uws application. This includes the dmocps control system
+  # application.
+  uws: false
+
   # -- Enable the vault-secrets-operator application. This is required for all
   # environments.
   vault-secrets-operator: true
 
   # -- Enable the vo-cutouts application
   vo-cutouts: false
+
+# -- Configuration particular to the control system
+# If this section is used, it can have the following parameters:
+# _appNamespace_ (Set the namespace for the control system components)
+# _imageTag_ (The image tag to use for control system images)
+# _siteTag_ (The tag that tells the control system component where it is running)
+# _topicName_ (The Kafka identifier for control system topics)
+# _kafkaBrokerAddress_ (The Kafka broker address)
+# _kafkaTopicReplicationFactor_ (The Kafka topic replication factor)
+# _schemaRegistryUrl_ (The Schema Registry URL)
+# _s3EndpointUrl_ (The S3 URL for the environment specific LFA)
+controlSystem: null
```

Comment:
This is where you should set all of the project-wide defaults, by replicating the same structure for `controlSystem` here. In other words, every one of these options should be set here with a default and a documentation string, and then the individual environments can override those options as needed.

This means there are some defaults exposed to environments that don't run a control system, but that's fine since presumably only the control system applications will use those defaults and none of them are enabled. The tricky part will be figuring how what to do with the things that must be set, since there's no reasonable default. The right approach is to add validators, but I don't think all of that work has to be done up-front before this is merged since it will be a lot easier to improve once we can get this landed.

Reply:
OK, that's what I had originally done, but took it out to hide things more. I put back what I originally did now.